### PR TITLE
Add Prepping for an Interview

### PR DESCRIPTION
**Pull request by memcmahon**

### CODE REVIEW

File: module6/lessons/Week5/PreppingForInterviews.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    * Launch students work on a combination of individual and group projects throughout the program.
+    * The curriculum and projects are designed for career-starters to give a broad understanding of web-development.
+
+<section class='notes' markdown='1'>
+
+Because 'Launch' is not a specific name for the skills you have learned, we recommend that you include your experience at Turing with more industry-standard wording.  For example:
+
+```
+School: Turing School
+Program: Web Development in C#/.Net
```
Comment (zoefarrell) [inline module6/lessons/Week5/PreppingForInterviews.md]: There were questions at Retro about what the official certificate will be for Launch. Skylar was especially interested and asked if she could have input. I encouraged her to send us her thoughts. 

Personally, I like Backend Development in C#/.NET better than Web Development. I think our students are far more prepared for backend questions than general web dev. What is your preference?

File: module6/lessons/Week5/PreppingForInterviews.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+# How to Talk about Turing
+
+You all have spent 9 months on this Turing journey, so *anything* you want to share about **your own experience** is great!  But, you might get some questions about Turing more generally - here are some key points that you can reference:
+
+* Turing School is an accredited, non-profit, fully-remote computer programming school, focused solely on helping students develop the technical and professional skills need for a  fulfilling career in technology.
+* All of Turing's programs are assessment-based; students continually prove their understanding through assessments.
+* Though not *always* on offer, [Launch](https://turing.edu/launch) is a program that teaches Web Development in C#/.NET with a focus on backend development.
+    * Our [curriculum](https://launch.turing.edu/) is open source and can be shared with anyone!
+    * Launch students work on a combination of individual and group projects throughout the program.
+    * The curriculum and projects are designed for career-starters to give a broad understanding of web-development.
```
Comment (zoefarrell) [inline module6/lessons/Week5/PreppingForInterviews.md]: Two other things unique to Launch I think are worth including. 

The PD focus is unique: Productive/Unproductive struggle, asking questions, and giving/receiving feedback.

The project where they jumped onto another team and contributed I think is also worth mentioning as a great very real world example.

File: module6/lessons/Week5/PreppingForInterviews.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+* Note-taking supplies
+* Fully charged computer
+* A few copies of your resume
+
+If you are interviewing remotely, make sure you have handy:
+* A copy of your resume
+* note-taking supplies
+* Projects that you want to highlight pulled up and ready to highlight!
+
+<section class='call-to-action' markdown='1'>
```
Comment (zoefarrell) [inline module6/lessons/Week5/PreppingForInterviews.md:88]: I like this activity!

I think makes most sense to move this section to the end. And put the talking about coding first. That way after introducing the activity we can break for independent work.

File: module6/lessons/Week5/PreppingForInterviews.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
+---
+layout: page
+title: Prepping for an Interview
+---
+
+You've busted your butt and landed an interview! GREAT! Now what?
+
+Each interview deserves its own preparation.  Try to give yourself at least a few days before the interview so that you are fully prepared and confident on the day.
+
+# Prepping for an Interview - A Checklist!
```
Comment (zoefarrell) [inline module6/lessons/Week5/PreppingForInterviews.md:34]: I think this list is great!

I'm not sure where to add this, but I think worth adding a note about it being totally ok to ask the recruiter what format of interview to expect. I've heard from a couple of students that last week they were expecting a different format (either a code challenge or all STAR). Apparently Isiah has been doing a lot of edibit practice over break after the disaster interview with you.

**Review by zoefarrell — APPROVED** [review]

File: module6/lessons/Week5/PreppingForInterviews.md (hunk 1 above)
Comment (memcmahon) [reply module6/lessons/Week5/PreppingForInterviews.md]: I'm good with Backend Development!

File: module6/lessons/Week5/PreppingForInterviews.md (hunk 3 above)
Comment (memcmahon) [reply module6/lessons/Week5/PreppingForInterviews.md:88]: I agree, reordering makes more sense!

File: module6/lessons/Week5/PreppingForInterviews.md (hunk 4 above)
Comment (memcmahon) [reply module6/lessons/Week5/PreppingForInterviews.md:34]: Glad that Isiah has been practicing!  

I'm a bit surprised that some students were caught off-guard 🙃 In our PD session on Monday, James asked what the Thursday interview would be like, and we even went over how you could incorporate STAR into a technical question 😆

Comment (memcmahon) [reply module6/lessons/Week5/PreppingForInterviews.md:34]: I'm going to add an initial step to confirm the interview-type!

**Pull request merged**